packages/composite-checkout/src/components/button.js

```diff
@@ -180,6 +208,29 @@ function getBackgroundColor( { buttonType, buttonState, theme } ) {
 	}
 }
 
+function getBackgroundAccentColor( { buttonState, buttonType, theme } ) {
```

Comment:
@fditrapani: this is pretty duplicative of `getRollOverColor` except for the variation in `disabled` behavior. I didn't know if I should add an accent variable for each state/type explicitly in theme.js. What do you think?

Reply:
What you have done seems to work fine. I wouldn't add anything if we don't need to.